src/includes/getting-started-config/ruby.delayed_job.mdx

```diff
@@ -3,6 +3,6 @@ Initialize the SDK:
 ```ruby
 Sentry.init do |config|
   config.dsn = '___PUBLIC_DSN___'
-  config.breadcrumbs_logger = [:sentry_logger]
+  config.breadcrumbs_logger = [:sentry_logger, :http_logger]
```

Comment:
@rhcarvalho If we're talking about `sentry_logger`, it really depends on what the libraries the user has. 
If the application already uses Rails and has `active_support_logger`, having `sentry_logger` would generally create noise. For example, my company's project doesn't use it.

But if the application does something like import/export operation with logging, having it could be useful. That's why I added it to the [rubyapi](https://github.com/rubyapi/rubyapi/pull/739) project.

As for `http_logger`, because it's a newly added logger and didn't exist in the old SDK. So I want to see how our users think about it before making it a default.